CHANGES.rst

```diff
@@ -14,6 +14,24 @@ Changelog
 
 .. towncrier release notes start
 
+1.5.0a0 (2020-01-07)
```

Comment:
You shouldn't change this file.

Reply:
Sorry, missed this when committing. I should simply revert this change, right?

Reply:
Yes, just make it go away :)